sdk/jobs/pipelines/1a_pipeline_with_components_from_yaml/pipeline_with_components_from_yaml.ipynb

```diff
@@ -61,10 +95,42 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "#Enter details of your AML workspace\n",
-    "subscription_id = '<SUBSCRIPTION_ID>'\n",
-    "resource_group = '<RESOURCE_GROUP>'\n",
-    "workspace = '<AML_WORKSPACE_NAME>'"
+    "config_path = \"../../../../.azureml/config.json\"\n",
```

Comment:
what's this path?

Reply:
Root level of this project, maybe add a comment?